spec/factories/comments_factory.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :comment do
+    title "MyString"
+    body "MyString"
+    Message nil
```

Comment:
yeah and generated wrong. I had capitalized Message in the scaffold on my first pass and manually changed it, and must have missed this one.